crates/tuono/src/cli.rs

```diff
@@ -47,12 +47,46 @@ fn init_tuono_folder(mode: Mode) -> std::io::Result<()> {
     Ok(())
 }
 
+fn check_ports(mode: Mode) -> std::io::Result<()> {
```

Comment:
Actually, this function doesn't return anything.

```suggestion
fn check_ports(mode: Mode) {
```